docs/src/release_notes/v1.21.md

```diff
@@ -6,16 +6,79 @@ For a complete list of changes, please refer to the
 [commits](https://github.com/cloudnative-pg/cloudnative-pg/commits/release-1.21)
 on the release branch in GitHub.
 
-## Version 1.21.5
+## Version 1.21.6
 
-**Release date:** Apr 24, 2024
+**Release date:** Jun 12, 2024
 
 !!! Warning
-    Version 1.21 is approaching its End-of-Life (EOL) on May 24, 2024.
-    If you haven't already, please begin planning for an upgrade promptly to
-    ensure continued support and security.
+    This is expected to be the last release in the 1.21.X series.
+    Users are encouraged to update to a newer minor version soon.
+
+### Enhancements:
+
+- Enabled configuration of standby-sensitive parameters during recovery from a
+  physical backup (#4564)
+
+- Enabled the configuration of the liveness probe timeout via the
+  `.spec.livenessProbeTimeout` option (#4719)
+
+- `cnpg` plugin for `kubectl`:
+
+    - Introduced support for ANSI colors in the plugin through the `--color`
+      option, which accepts `always`, `never`, and `auto` (default) as values
+      (#4775)
+    - The plugin is now available on Homebrew for macOS users (#4602)
+
```

Comment:
This PR is only backported to 1.23, no 1.21 nor 1.22. Do we still want it in the RN since the plugin is built on 1.23?

Reply:
Homebrew always delivers the latest version, and that version is retrocompatible. I would keep it.